store/tikv/error.go

```diff
@@ -22,6 +22,8 @@ import (
 var (
 	// ErrBodyMissing response body is missing error
 	ErrBodyMissing = errors.New("response body is missing")
+	// When TiDB is closing and send request to tikv fail, do not retry, return this error.
+	errTiDBMoribid = errors.New("tidb server moribid")
```

Comment:
Will this error message be returned to the client?
If so, should we make it more readable for the users?

Reply:
This state is set in `closeDomainAndStorage` which is called after `TryGracefulDown`:
https://github.com/pingcap/tidb/pull/10511/files#diff-af2c9279ed41f7023f9b6b73cba2d18fR582

After `TryGracefulDown`, connections to client have already been closed, so the error doesn't return to the client.